source/extensions/load_balancing_policies/maglev/maglev_lb.cc

```diff
@@ -2,6 +2,8 @@
 
 #include "envoy/config/cluster/v3/cluster.pb.h"
 
+#include "source/common/runtime/runtime_features.h"
```

Comment:
why the runtime include?

Reply:
We removed the directly dependency to the `upstream_lib` and `upstream_impl.h` in the `maglev_lb.h`. That change result in we must include the `runtime_features.h` manually.
I think this change is reasonable and make the dependency graph simlper.